### ISSUE TITLE
BUG: 'SCMP_MAJOR_VERSION' undeclared

### ISSUE THREAD

**Issue by Freeaqingme** [seccomp/libseccomp-golang#23]
When I use the latest commit, I get these errors while building my application. If I use e3496e3a417d1dc9ecdceca5af2513271fed37a0 there's no problem whatsoever.

<pre>github.com/seccomp/libseccomp-golang
src/github.com/seccomp/libseccomp-golang/seccomp_internal.go: In function 'get_major_version':
src/github.com/seccomp/libseccomp-golang/seccomp_internal.go:141:29: error: 'SCMP_MAJOR_VERSION' undeclared (first use in this function)
        return (unsigned int)SCMP_MAJOR_VERSION;
                             ^
src/github.com/seccomp/libseccomp-golang/seccomp_internal.go:141:29: note: each undeclared identifier is reported only once for each function it appears in
src/github.com/seccomp/libseccomp-golang/seccomp_internal.go: In function 'get_minor_version':
src/github.com/seccomp/libseccomp-golang/seccomp_internal.go:146:29: error: 'SCMP_MINOR_VERSION' undeclared (first use in this function)
        return (unsigned int)SCMP_MINOR_VERSION;
                             ^
src/github.com/seccomp/libseccomp-golang/seccomp_internal.go: In function 'get_micro_version':
src/github.com/seccomp/libseccomp-golang/seccomp_internal.go:151:29: error: 'SCMP_MICRO_VERSION' undeclared (first use in this function)
        return (unsigned int)SCMP_MICRO_VERSION;
                             ^
Makefile:25: recipe for target 'binary' failed
make: *** [binary] Error 2</pre>

**Comment by mheon** [comment]
I have a patch which (should) fix this - can you pull from https://github.com/mheon/libseccomp-golang branch tsync_and_bugfix and verify that the issue is fixed?

**Comment by Freeaqingme** [comment]
Thanks @mheon using that branch fixes the compilation problem, so it would seem like an appropriate fix for this issue.

I have not, however, been able to test it entirely (not on e3496e3 either). For some reason the call to C.seccomp_load(f.filterCtx) blocks and appears to hang indefinitely. I guess that warrants a separate issue(?).

**Comment by mheon** [comment]
@Freeaqingme Yeah, a new issue sounds appropriate for that. I'll see about getting the bugfix patches merged on Monday, and close this issue when they've landed. The `seccomp_load` issue seems separate and should be tracked elsewhere. When you create the issue, can you give some details of your environment (distribution, kernel version, CPU architecture)? Thanks in advance.

**Comment by mheon** [comment]
Fixed by 84e90a91acea0f4e51e62bc1a75de18b1fc0790f
